junebug/logging_service.py

```diff
@@ -111,6 +112,9 @@ def __init__(self, worker_id, path, rotate, max_files, logger=None):
         self.max_files = max_files
 
     def startService(self):
+
+        if not os.path.exists(os.path.dirname(self.path)):
+            os.makedirs(os.path.dirname(self.path),0777)
```

Comment:
PEP8 likes a space after a `,`.

GitHub seems to have highlighted the `0777` in red. I'm not entirely sure why. One can also use `0o777` for octal literals though, which is perhaps a little more explicit and maybe GitHub will not highlight in red. Up to you. :)